packages/zk-transfer-web/src/components/Token.tsx

```diff
@@ -136,4 +167,4 @@ export const TokenCard = (props: any) => {
   );
 };
 
-export default TokenInput;
+export default React.memo(TokenInput);
```

Comment:
The `<TokenInput>` component renders everytime the cursor overs on them. So the `React.memo` wrapper will memoize (cache) the component. This will reduce the number of times the component gets to re-render